app-backend/api-it/src/test/resources/endpoint-scopes.csv

```diff
@@ -1,205 +1,185 @@
 Path,Domain:Action,Verb

```

Comment:
Some of the paths here simply didn't exist. A few needed different scopes to work - generally I assumed what was in the API was correct and updated the required scopes here accordingly.